Rakefile

```diff
@@ -459,7 +461,15 @@ task :ci do
       sh 'bundle exec appraisal contrib rake spec:shoryuken'
       sh 'bundle exec appraisal contrib rake spec:sinatra'
       # Rails minitests
-      sh 'bundle exec rake benchmark'
+      sh 'bundle exec appraisal rails5-mysql2 rake test:rails'
```

Comment:
Also might be worthwhile adding a brief comment explaining that we don't test Rails 3/4 here because they require Bundler < 2.0, just so if we're playing around in here later we remember why they are missing.

Reply:
Added!